### ISSUE TITLE
NIC details can escape bounding box

### ISSUE THREAD

**Issue by dankamongmen** [dankamongmen/omphalos#32]
On my Debian workstation, I'm seeing the following:

```
    ╭─press 'v' to dismiss details─────────────────────────────────────────────╮
    │r8169            04:d9:f5:7e:39:02                                        │
    │Realtek Semiconductor Co., Ltd. RTL8111/8168/8411 PCI Express Gigabit Ether
    net Controller? GSO- GRO+ LRO- TCsm- RCsm+ TVln+ RVln+ MTU: 1500           │
  M │TXfd: 14   flen: 2048   fnum: 512  blen: 4Ki   bnum: 256   txr:   1MiB    │
  M │Tbyte: 19762                frames: 178                  aborts: 0        │
  M │RXfd: 15   flen: 32768  fnum: 4Ki  blen: 32Ki  bnum: 4096  rxr: 128MiB    │
    │Rbyte: 1162703              frames: 7322                                  │
  M │mform: 0                    noprot: 31                                    │
  M │drops: 0                    truncs: 0                    (0 recov)        │
    ╰───────────────────────https://nick-black.com/dankwiki/index.php/Omphalos─╯
```

Make sure that NIC name is properly bounded.

**Comment by dankamongmen** [comment]
Outstanding PR #45 improves upon this; the subsequent line is no longer affected, but we do still blow out the right border on the offending line.

**Comment by dankamongmen** [comment]
resolved for 0.99.13.
